releasenotes/notes/support-scheduler-in-execute-42a1ebe9e3a67bbb.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Circuits can be scheduled during ``execute`` by setting ``schedule_circuit=True``.
+    For example::
+
+        job = execute(qc, backend, schedule_circuit=True, shots=4321)
```

Comment:
`shots` isn't relevant to this feature, so maybe show `scheduling_method` instead, since that's a kwarg users will most likely need if they use this feature

Reply:
fixed in [1f6e5cc](https://github.com/Qiskit/qiskit-terra/pull/3792/commits/1f6e5cc4cca5c30720f37aa7496fbdaf33d317fd). :)